src/components/MyList/MyList.tsx

```diff
@@ -0,0 +1,34 @@
+import { memo } from 'react';
+import { User } from '../../models/user';
+import { MyListItem } from '../MyListItem/MyListItem';
+import './MyList.css';
+
+interface MyListProps {
+  users: User[];
+}
+
+function MyList(props: MyListProps) {
+  const { users } = props;
+
+  console.log('render list');
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td className="header">Is Active</td>
+          <td className="header">Name</td>
+          <td className="header">Age</td>
+          <td className="header">Organization</td>
+        </tr>
+      </thead>
+      <tbody>
+        {users.map((user, index) => {
+          const isLast = index === users.length - 1;
+          return <MyListItem key={index} user={user} isLast={isLast} />;
```

Comment:
Never never never use the index as the key. This can negatively impact performance and may cause issues with the component state. Always prefer a unique prop related to the object that you are interating, eg: user's id

- https://react.dev/learn/rendering-lists#why-does-react-need-keys
- https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern

And this applies to all frameworks, no exception, Jetpack, Swift, Flutter, etc

Reply:
done, I'll use name for the key, I don't have a unique identifier for my data but I'll treat the name as one for this project